lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -30,15 +46,31 @@ class ChangesPayloadGenerator
       def initialize(opts = {})
         @original = opts.fetch(:original, {}) || {}
         @updated = opts.fetch(:updated, {}) || {}
+        @translated_attrs = opts.fetch(:translated_attrs, {}) || {}
       end
 
       def execute
+        translate_attrs if translated_attrs.any?
```

Comment:
It feels wrong to me that it's changing the object instead of creating an updated copy. Not thread safe.